lib/fxpay/receipts.js

```diff
@@ -500,4 +500,40 @@
     return atob(s);
   }
 
+
+  function ReceiptInfo(receipt, verificationResult) {
+    // Information about a receipt.
+
+    // This is the original JWT (JSON Web Token) receipt string.
```

Comment:
Yeah, I also want to make an object like this for ProductInfo which is documented on MDN but not really in the code. I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1143024